freeipa/src/main/java/com/sequenceiq/freeipa/service/stack/FreeIpaHealthDetailsService.java

```diff
@@ -45,6 +45,10 @@
 
     private static final Pattern NEW_NODE_PATTERN = Pattern.compile("Check connection from master to remote replica '(.[^\']*)");
 
+    private static final String WRONG_MASTER_MESSAGE = "invalid 'cn': must be";
+
+    private static final int MAX_RETRIES = 3;
```

Comment:
why 3?
also it should be configurable

Reply:
Made configurable.  I made it three in case we're in the middle of a replication change or something else that switches the value.